Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/SelectorsPollerScript.js

```diff
@@ -489,33 +491,33 @@ window.__firefox__.execute(function($) {
     }
     // Find selectors we have a rule index for
     const rulesToRemove = Array.from(selectors)
-      .map(function (selector) { return CC.allSelectorsToRules.get(selector) })
+      .map(function (selector) { return CC.allHideSelectorsToRules.get(selector) })
```

Comment:
nit, but maybe standardize on `function` or `=>` (here the `=>` is probably better, since you don't need `this`)

Reply:
Fixed. Well actually its been removed